a4kSubtitles/services/podnadpisi.py

```diff
@@ -74,7 +74,8 @@ def map_result(result):
             'action_args': {
                 'url': '%s%s' % (__url, result['download']),
                 'lang': lang,
-                'filename': name
+                'filename': name,
+                "episodeid": meta.episode.zfill(3) if meta.is_tvshow and meta.episode else "",
```

Comment:
Nice find. Sub packs were planned at one time that is why there was some code to support them, but was left unfinished and forgotten.

Is podnadpisi, subdl and subsource the only one supporting packs?
Even so, it shouldn't be an issue to add the same code for all? We could just add it to `search.py/__add_results` (meta needs to be passed trough `__complete_search`).

Reply:
> Is podnadpisi, subdl and subsource the only one supporting packs?

I don't know about other the only one works with my language were subdl, subsources and opensubtitles.

> it shouldn't be an issue to add the same code for all?

I don't think so it should be fine, 
I'll add it as a default value in case if it possible to detect episode via service it can be overwritten from service module.